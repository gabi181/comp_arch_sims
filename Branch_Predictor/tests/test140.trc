1 8 8 0 local_history global_tables not_using_share
0x7D1221A4 T 0x6180C380
0x21D1A0D8 T 0x53227708
