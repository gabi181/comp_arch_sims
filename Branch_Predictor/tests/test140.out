0x7d1221a4 N 0x7d1221a8
0x21d1a0d8 N 0x21d1a0dc
flush_num: 2, br_num: 2, size: 559b
